src/org/zaproxy/zap/extension/alert/ExtensionAlert.java

```diff
@@ -74,6 +74,7 @@
 public class ExtensionAlert extends ExtensionAdaptor implements 
 		SessionChangedListener, XmlReporterExtension, OptionsChangedListener {
 
+    public static final String UI_NAME = Constant.messages.getString("alerts.name");

```

Comment:
Thanks, sent the PR as I ran out of the house, didn't even notice the failure .... will tweak in a bit when I'm home again. Thanks thc202